lldb/source/Plugins/Language/Swift/SwiftLanguage.cpp

```diff
@@ -1832,19 +1832,110 @@ SwiftLanguage::GetDemangledFunctionNameWithoutArguments(Mangled mangled) const {
   return mangled_name;
 }
 
-void SwiftLanguage::FilterForLineBreakpoints(
-    llvm::SmallVectorImpl<SymbolContext> &sc_list) const {
-  llvm::erase_if(sc_list, [](const SymbolContext &sc) {
-    // If we don't have a function, conservatively keep this sc.
-    if (!sc.function)
-      return false;
+namespace {
+using namespace swift::Demangle;
+struct AsyncInfo {
+  const Function *function;
+  NodePointer demangle_node;
+  std::optional<uint64_t> funclet_number;
+
+  /// Helper function for logging.
+  std::string to_str() const {
```

Comment:
The more idiomatic thing in LLVM is to add a free-standing toString() function.

Reply:
which I think will make it work with operator<< as well

Reply:
good point!